packages/app/src/systems/Network/services/network.ts

```diff
@@ -53,7 +53,7 @@ export class NetworkService {
         ...(count === 0 && { isSelected: true }),
         id: uniqueId(),
       });
-      return db.networks.get(id);
+      return db.networks.get(id) as Promise<Network>;
```

Comment:
Why do we type cast here with `as Promise<Network>` as well as in other places?

Reply:
This is because the return can be undefined by default, but on this cases it never returns undefined as it was just add.